warehouse_pipeline/schema.sql

```diff
@@ -29,8 +29,15 @@ CREATE TABLE IF NOT EXISTS botanist (
 );
 
 
-CREATE TABLE plant (
+CREATE TABLE IF NOT EXISTS species (
     species_id SERIAL PRIMARY KEY,
+    scientific_name text NOT NULL UNIQUE
+    
+);
+
```

Comment:
Very minor change, but you could consider either changing all spacing between statements to be a single line, or increase this one to two lines.